### ISSUE TITLE
Calling .wait() or .close() on failed job (at .closed state) causes an exception

### ISSUE THREAD

**Issue by pryg-skok** [aio-libs/aiojobs#29]
Hi,

thanks for your library!
Trying to figure out how to forward exceptions from spawned jobs into main code and come against non correct behavior. If job was failed (raised and exception inside) and its state was set to closed and `job.wait()` called on it, the exception raised:
```Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/aiohttp/web_protocol.py", line 381, in start
    resp = await self._request_handler(request)
  File "/usr/local/lib/python3.6/site-packages/aiohttp/web_app.py", line 310, in _handle
    resp = await handler(request)
  File "test.py", line 76, in check
    print(await job.wait())
  File "/usr/local/lib/python3.6/site-packages/aiojobs/_job.py", line 62, in wait
    await self._close(scheduler.close_timeout)
AttributeError: 'NoneType' object has no attribute 'close_timeout'
```

It seems there should be either different behavior like idempotent jobs waiting, or maybe simply forbid to wait closed jobs.

**Comment by asvetlov** [comment]
Thanks for report.
The bug should be fixed, I'll take a look

**Comment by spumer** [comment]
Please, fix it

Simple test to reproduce
```python
async def test_wait_failed(event_loop):
    scheduler = await _create_scheduler(loop=event_loop)

    async def coro(res):
        if isinstance(res, Exception):
            raise res
        return res

    expected = [1, RuntimeError()]
    jobs = [await scheduler.spawn(coro(x)) for x in expected]

    with pytest.raises(RuntimeError):
        await asyncio.gather(*(j.wait() for j in jobs), loop=event_loop)

    await jobs[-1].wait()
```

**Comment by asvetlov** [comment]
Fixed by #64
